client/src/lib/ReactRouteRegister.js

```diff
@@ -170,7 +170,7 @@ class ReactRouteRegister {
  * `reactRouteRegister`, the copy of `reactRouteRegister` in `framework` is not
  * the same copy of `reactRouteRegister` available in `custom`.
  *
- * @TODO Look into SystemJS as a solution https://github.com/systemjs/systemjs
+ * @link Look into SystemJS as a solution https://github.com/systemjs/systemjs
```

Comment:
```suggestion
```
Just remove the todo comment outright please

Reply:
Updated